### DISCUSSION TITLE
Configure Dependabot to only create PRs for minor or patch for security updates

### DISCUSSION THREAD

**Discussion by zijianzz** [community/community#138913] [New to GitHub]
I'd like to know is there any possibility to configure dependabot to take only minor and patch update for security updates? I know it's possible for version update by using `ignore.update-types` but nothing seems exist for security update from what I saw. Thanks in advance for your help!

**Comment by sartizalamayon** [comment]
Dependabot does not have a direct configuration option to restrict security updates to only minor and patch versions. The ignore.update-types feature you mentioned is specifically for version updates, but it does not apply to security updates, which always attempt to apply the most relevant security fixes regardless of version changes. You can manually review security PRs. If you need more control, you could explore using custom GitHub actions or CI scripts that check the Dependabot PR for a security update and inspect whether it's a minor or patch update. Based on that, you could automatically approve or reject it.
